gen/atlas.tmpl

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches:
       - {{ .DefaultBranch }}
+    paths:
+      - .github/workflows/ci-atlas.yaml
+      - '{{ .Path }}/*'
```

Comment:
shouldn't these paths only be inside the `pull_request` and `master branch` triggers?